src/ServerTelemetryChannel/Implementation/ApplicationStoppingEventArgs.cs

```diff
@@ -32,7 +27,7 @@ public async void Run(Func<Task> asyncMethod)
         {
             try
             {
-                await this.asyncMethodRunner(asyncMethod);
+                await this.asyncMethodRunner(asyncMethod).ConfigureAwait(false);
```

Comment:
If there is a chance that any code expected that continuation will happen on the same thread, it may be broken now - "false" will allow to continue on the different thread and any thread-related context will be lost.

It's OK to have with change with several test runs and a pilot deployment to spot issues early if any. @SergeyKanzhelev  may know right away if we had any such context-depending code, so he can clear this quickly.

Edit: I noticed similar change in 3 other places as well.